server/db/db.js

```diff
@@ -50,7 +50,7 @@ function createSensorsVitalFromRow(r, allLocations) {
   return new SensorsVital(r.id, r.missed_door_messages, r.is_door_battery_low, r.door_last_seen_at, r.reset_reason, r.state_transitions, r.created_at, r.is_tampered, location)
 }
 
-async function beginTransaction(retryCount = 0) {
+async function runBeginTransactionWithRetries(retryCount) {
```

Comment:
Can you explain the rationale behind renaming the function from `beginTransaction` to `runBeginTransactionWithRetries`? I couldn't find any instances where `runBeginTransactionWithRetries` is updated with the new name in files like BraveAlerterConfiguration.js and vitals.js, which depend on it.